scripts/CompareFitMinimizers/Definitions/color_definitions.txt

```diff
@@ -0,0 +1,22 @@
+.. .. these are for the color scale "goodness of fit"
+.. role:: ranking-top-1
+.. role:: ranking-top-2
+.. role:: ranking-med-3
+.. role:: ranking-low-4
+.. role:: ranking-low-5
+
+
+.. .. This is for the ranking/gradient colors in the table cells. The function changes the class
+      of the parents of the text/numbers that go in some table cells (as span elements), so that the cells themselves
+      (parents) are assigned a certain style class (background color)
+
+.. raw:: html
```

Comment:
Could you investigate if the content of *.js file could be included in the ..raw:: html? If this can be done it would mean the html files can be generated and located anywhere